Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Dwarven Ethnicity Rework

## Changelog:
- Reworked the old ethnicity used by Dwarves to be closer to how they are depicted in most Warcraft media.
- Greater focus on making female dwarves look prettier. 😳


# How to test:
1. Open up Ruler Designer and generate male and female **_Dwarves_** using the reworked ethnicities (`dwarven_rework`, `frostborn_rework`, `bronzebeard_rework`, `dark_iron_rework`, `wildhammer_rework`) while comparing them to the old ethnicities higher up on the list of ethnicities.
2. You may also load up a game and look for **_Dwarves_** in-game to see how they look when animated.

Provide feedback on the reworked ethnicity and whether it's an improvement over the old or not.
